layouts/partials/announcement_banner/announcement_banner.html

```diff
@@ -4,4 +4,11 @@
     <a href="{{.Site.Params.corp_url}}{{$announce.link}}">{{$announce.text | safeHTML }}</a>
     <i class="icon icon-small-x pull-right"></i>
 </div>
+<script>
+    var announcement_banner;
```

Comment:
Let's change this. The banner by default is hidden with css which causes it to display then hide.

Remove this scss. The banner should be hidden if local/session storage have it set to be hidden. See corpsites method for checking

.announcement_banner {
    display: none;
}